widget/embedded/src/QueueManager.tsx

```diff
@@ -91,14 +92,16 @@ function QueueManager(props: PropsWithChildren<{ apiKey?: string }>) {
     isMobileWallet,
   };
 
+  const isActiveTab = useUiStore.use.isActiveTab();
```

Comment:
Please check your PR on firefox.
My test env:
https://www.mozilla.org/en-US/firefox/116.0/releasenotes/?utm_source=firefox-browser&utm_medium=firefox-desktop&utm_campaign=about-dialog

Problem:
Active tab is not detected correctly. (we have multiple active tabs)